website/docs/index.md

```diff
@@ -53,6 +53,41 @@ ln -s -f ../../scripts/pre-commit .git/hooks/pre-commit
 ```
 
 
+## How to Create a Good UNIX Interface
+
+One of our goals with this project is learning how to write programs and, more importantly,
+the importance of not writing code vs. writing code. 
+
+Here are some 
+- [good rules for writing good unix programs](https://lasr.cs.ucla.edu/ficus-members/geoff/interfaces.html)
+
+A good example can be seen in notions of progress bars. The idea of progress bars has come up twice now, from 
+two different contributors, once in cp, and once in dd.
```

Comment:
@rjoleary there was a discussion two years ago about the `progress bar` thing, I'd like to add that, in that time we were focused in implementing the userland of u-root using POSIX as a reference, and if I remember correctly there were no such thing as progress bar option. But I think a new discussion must be risen in order to address this issue.